vim25/types/types.go

```diff
@@ -8231,7 +8231,7 @@ func init() {
 type ClusterDasAdmissionControlPolicy struct {
 	DynamicData
 
-	ResourceReductionToToleratePercent int32 `xml:"resourceReductionToToleratePercent,omitempty"`
+	ResourceReductionToToleratePercent *int32 `xml:"resourceReductionToToleratePercent"`
```

Comment:
if you still want to let users skip that value it is better to return `omitempty` back. If it is not there the value will be mandatory.
ResourceReductionToToleratePercent *int32 `xml:"resourceReductionToToleratePercent,omitempty"`

Reply:
That's not the case with the xml encoder, see: https://golang.org/src/encoding/xml/marshal.go?s=3363:3406#L69

> or, if the pointer is nil, by writing nothing

Reply:
that will do! 
I would keep `omitempty` just for the simplicity so users would not need to go to the marshaller docs.

Reply:
Sure, pushed a new commit that does so.